client/components/payment-method/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { PropTypes } from 'react';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import { sprintf } from 'sprintf-js';
+import { translate as __ } from 'lib/mixins/i18n';
+
+// TODO - create PaymentMethod like https://github.com/Automattic/wp-calypso/blob/29984e5677960bdba231577a253cf864d4f17796/client/my-sites/upgrades/checkout/stored-card.jsx
+// or https://github.com/Automattic/wp-calypso/tree/29984e5677960bdba231577a253cf864d4f17796/client/my-sites/upgrades/checkout
+
+const PaymentMethod = ( { value, currentValue, onChange, name, cardType, cardDigits, expiry } ) => {
+	const cardClasses = 'payment-method ' + cardType.toLowerCase();
+	const expirationDate = sprintf( __( 'Expires %(monthAndYear)s' ),
+		{
+			monthAndYear: expiry,
+		}
+	);
+
+	return (
+		<FormLabel className="payment-method__label">
+			<FormRadio
+				className="payment-method__radio"
+				value={ value }
+				checked={ value === currentValue }
+				onChange={ () => onChange( value ) }
```

Comment:
The `PaymentMethodSelection` `onChange` property is ignoring its argument, so you can just do here:
`onChange={ onChange }`